client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -13,9 +17,12 @@ import NavItem from 'components/section-nav/item';
 import FollowersCount from 'blocks/followers-count';
 import SegmentedControl from 'components/segmented-control';
 import QueryJetpackPlugins from 'components/data/query-jetpack-plugins';
+import { isPluginActive } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { UNITS as StoreStatsTabs } from 'extensions/woocommerce/app/store-stats/constants';
```

Comment:
@sirreal: Is it intentional that a file in Calypso's core is requiring an extension? Is there a plan to eliminate this? I don't think it's a precedent we want to set.

FWIW there is prior art for _component decoration_ in Calypso's first concept extensions PR:

https://github.com/Automattic/wp-calypso/pull/8103/files

^ a search for `decorat` is a good starting point.